srcpkgs/jmeter/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'jmeter'
+pkgname=jmeter
```

Comment:
This should be `jmeter-bin` as it's installing binaries and not building from source.